### DISCUSSION TITLE
Wallpaper recommendations?

### DISCUSSION THREAD

**Discussion by whazor** [nix-community/stylix#423] [Q&A]
I have found it difficult to find a nice wallpaper that results in a color scheme that I like for programming. So I wonder if anyone has a nice recommendation for a wallpaper that results in a nice color scheme. Thanks!

**Comment by danth** [comment]
I have a couple of example configs [here](https://gist.github.com/danth) that you could try. Some are better than others

**Comment by trueNAHO** [comment]
> I have found it difficult to find a nice wallpaper that results in a color scheme that I like for programming.

You could declare your wallpaper while enforcing a matching color scheme independently.
